composeApp/build.gradle.kts

```diff
@@ -61,6 +61,10 @@ kotlin {
             implementation(libs.compose.webview.multiplatform)
             implementation(libs.richeditor.compose)
         }
+        commonTest.dependencies {
+            implementation(libs.kotlin.test)
+            implementation("app.cash.turbine:turbine:0.7.0")
```

Comment:
You haven't used implementation("app.cash.turbine:turbine:0.7.0") in your changes; it can be added when needed.